app/Stores/EntityStore.js

```diff
@@ -195,10 +230,18 @@ class EntityStore extends EventEmitter {
         this.emit('entries_loaded');
     }
 
+    emitDelete() {
+        this.emit('entries_deleted');
+    }
+
     addChangeListener(callback) {
         this.on('entries_loaded', callback);
     }
 
+    addDeleteListener(callback) {
+        this.on('entries_deleted', callback);
+    }
+
     removeChangeListener() {
```

Comment:
Not really related to this PR, but maybe we could call `EntityStore.removeAllListeners()` instead of this one, as you do in this PR, in the `Delete` component.